grails-app/init/specimenbrowser/BootStrap.groovy

```diff
@@ -1,3 +1,5 @@
+package specimenbrowser
```

Comment:
not `package au.org.ala.specimenbrowser`?

Reply:
This seems to be OK. There are a number of points in grails where it wants to use the app name as a package for configuration-type stuff and I figure it's best left alone. UrlMappings is another one.